.prettierrc.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
   parser: 'babel-flow',
   singleQuote: true,
   trailingComma: 'all',
+  arrowParens: 'avoid',
```

Comment:
commit-message comment:

    That commit reports that the default behavior before Prettier 2.0 is
    like setting `arrowParens: 'avoid'`. It's looking unlikely that we
    have an easy path to using Prettier 2.0 (see
    prettier/prettier-eslint-cli#304), but we might as well follow the
    RN template app.

This looks like a previous draft of the same material that's in the rest of the commit message. Probably you meant to delete this version?

Reply:
Thanks!